doc/releases/reef.rst

```diff
@@ -0,0 +1,281 @@
+
+.. important::
+
+   We are unable to build Ceph on Debian stable (bookworm) for the 18.2.0
+   release because of Debian bug
+   https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030129. We will build as
+   soon as this bug is resolved in Debian stable.
+
+   *last updated 2023 Aug 04*
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+* The ``AT_NO_ATTR_SYNC`` macro has been deprecated in favor of the standard
+  ``AT_STATX_DONT_SYNC`` macro. The ``AT_NO_ATTR_SYNC`` macro will be removed
+  in the future.
+
+Dashboard
+~~~~~~~~~
+
+* There is a new Dashboard page with improved layout. Active alerts
+  and some important charts are now displayed inside cards.
+
+* Cephx Auth Management: There is a new section dedicated to listing and
+  managing Ceph cluster users.
+  
+* RGW Server Side Encryption: The SSE-S3 and KMS encryption of rgw buckets can
+  now be configured at the time of bucket creation.
+
+* RBD Snapshot mirroring: Snapshot mirroring can now be configured through UI.
+  Snapshots can now be scheduled.
+  
+* 1-Click OSD Creation Wizard: OSD creation has been broken into 3 options:
+
+  #. Cost/Capacity Optimized: Use all HDDs
+
+  #. Throughput Optmized: Combine HDDs and SSDs
+
+  #. IOPS Optimized: Use all NVMes
+
+  The current OSD-creation form has been moved to the Advanced section.
+
+* Centralized Logging: There is now a view that collects all the logs from
+  the Ceph cluster.
+
+* Accessibility WCAG-AA: Dashboard is WCAG 2.1 level A compliant and therefore
+  improved for blind and visually impaired Ceph users.
+
+* Monitoring & Alerting
+
+      * Ceph-exporter: Now the performance metrics for Ceph daemons are
+        exported by ceph-exporter, which deploys on each daemon rather than
+        using prometheus exporter. This will reduce performance bottlenecks.
+
+      * Monitoring stacks updated
+
+      * Prometheus 2.43.0
+
+      * Node-exporter 1.5.0
+
+      * Grafana 9.4.7
+
+      * Alertmanager 0.25.0
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule manager module now retains one snapshot
+  less than the number mentioned against the config option
+  ``mds_max_snaps_per_dir``. This means that a new snapshot can be created and
+  retained during the next schedule run.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+  
+RADOS
+~~~~~
+
+* Trimming of PGLog dups is now controlled by the size rather than the version.
```

Comment:
@zdover23 can you add this bullet point above this line:

- RockDB is upgraded to version 7.9.2, which incorporates several performance improvements and features. This is the first release to have the ability to tune RockDB settings per column family, which allows for more granular tunings to be applied to different kinds of data stored in RocksDB. A new set of settings has been used to optimize performance for most kinds of workloads with a slight penalty in some cases, outweighed by large improvements in use cases such as RGW, in terms of compactions and write amplification.

Reply:
I'm going to assume s/RockDB/RocksDB/ and make that change when I add this. Speak now or hold your peace until you notice it later, at which point I will change it to whatever you like.

Reply:
This has been added.

Reply:
Yes, thanks, that was a typo

Reply:
Qapla.